docs/usage-manual.template.md

```diff
@@ -314,6 +318,24 @@ Running `$reframe eject browser` ejects the following code.
 !INLINE ../plugins/browser/browserEntry.js
 ~~~
 
+At that point you can link to a frontend library present in your project. Such a library is usually
```

Comment:
Thinking we could keep an abstract code snippet here while refering to the semantic ui use case.

Smth like

```js
import browserConfig from '@brillout/browser-config';

import './thirdparty/awesome-frontend-lib.js';
import './thirdparty/awesome-frotnend-lib.css';

initBrowser();

async function initBrowser() {
    await browserConfig.hydratePage();
}
```

Reply:
good idea, I'll do that